live-build/misc/upgrade-scripts/verify

```diff
@@ -41,6 +41,20 @@ function cleanup() {
 	#
 	[[ -z "$CONTAINER" ]] && return
 
+	#
+	# We need to collect logs within the container and move to
+	# container's host otherwise those will delete when container
+	# is deleted. We leverage support_info script to do the collection
+	# and only collect "log" and "dropbox" types. User who wish to add
+	# more collection types from support_info here should check if
+	# they work in the container.
+	# We collect only one copy to limit space usage.
+	#
+	report_progress 70 "Collecting verification logs"
+	rm -rf "${UPDATE_VERIFY_LOG_DIR:?}/*"
+	mkdir -p "${UPDATE_VERIFY_LOG_DIR}"
+	${SUPPORT_INFO_SCRIPT} -o "${UPDATE_VERIFY_LOG_DIR}" -t "log dropbox"
```

Comment:
This is supposed to run inside of the upgrade container, right? If so, I don't think this is correct.. right here, we'll be running on the host, not the container.

Reply:
I think it might be more appropriate to add this logic to the `verify-impl` script, which is intended to run inside the container.

Putting it in that script, will also make sure we run this log collection for "inplace" upgrades, as well as "not-inplace" upgrades. That script is also verify "app-gate" specific, so we wouldn't be leaking "app-gate" abstractions any more than we already do (e.g. leaking app-gate specific code into the upgrade scripts, which should be able to be run without the app-gate being installed).

What do you think?